canonical-web-and-design/ubuntu.com
# Build MicroCerts page

## Done

- Build microcerts hero based on [design](https://app.zeplin.io/project/5fc7a466de658e0cb99276c8/screen/5fd38d07a48d6b903ffb4e1f) at `/cube/microcerts`
- Fixed table column widths
- Hard coded adding 2 passed certifications to see contrast for design review (line 24 and 31 of `/cube/microcerts`)
- Fixed table view on mobile 

## QA

- Check page at: https://ubuntu-com-8951.demos.haus/cube/microcerts
- Check against [design](https://app.zeplin.io/project/5fc7a466de658e0cb99276c8/screen/5fd38d07a48d6b903ffb4e1f)
- Check SCSS and give any feedback on class names 


## Issue / Card

Fixes [#3507](https://github.com/canonical-web-and-design/web-squad/issues/3507)

## Screenshots

![image](https://user-images.githubusercontent.com/58959073/103756304-15764e00-5007-11eb-992d-8516db6697e3.png)
